app/views/test_executions/_execution_results.html.erb

```diff
@@ -8,20 +8,17 @@
 
 
 <% if execution.incomplete? %>
-  <h4>
-    <%= "PENDING: #{execution.created_at.to_time.strftime('%b %d, %Y at %I:%M:%S %p (%A)')}" %>
-  </h4>
+  <h4><i class="fa fa fa-circle-o-notch fa-spin inline-icon info-icon"></i>In Progress</h4>
+  <!-- <p>You do not need to reload your browser. Results will automatically display when the tests are done running. This should take about ___ seconds.</p> -->
+  <p>Wait 10 seconds, then reload your browser.</p>
+<% elsif execution.passing? %>
+  <h4><i class="fa fa-check-circle inline-icon success-icon"></i>Passed</h4>
```

Comment:
You can probably use Bootstrap's `text-success` class in place of `success-icon`